### ISSUE TITLE
DRAFT: Add facility list name and description to free text search

### ISSUE THREAD

**Issue by jwalgran** [open-apparel-registry/open-apparel-registry#624]
# Draft

This is a draft issue, and it will need to clarify specs and formalize what
counts as "done" before work begins on it.

Check with the issue's creator before starting work on this issue.

## Overview

Facilities have a `created_from` that points back to a `FacilityListItem`. From there we can get to the `FacilityList` and incorporate the list name and description into the free text search.

### Additional Context

Optional. Add any other context, screenshots, gifs, etc about the feature request here.

**Comment by kellyi** [comment]
I think we wrote this and #625 simultaneously! I added the snippet from the overview to the other issue, so I'm going to close this one.

**Comment by jwalgran** [comment]
Superseded by #625
